### PR TITLE
Adding mMARCO dataset

### PR DESCRIPTION
**Pull request by lhbonifacio**
Adding mMARCO (v1.1) to HF datasets.

### CODE REVIEW

File: datasets/mmarco/mmarco.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+    language: str = "english"
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for MS MARCO
+
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(MsMarcoConfig, self).__init__(**kwargs)
```
Comment (albertvillanova) [inline datasets/mmarco/mmarco.py:55]: No need to create a specific `datasets.BuilderConfig` subclass if you are not creating additional config attributes, others than name, description, version,...

File: datasets/mmarco/mmarco.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+
+class MsMarco(datasets.GeneratorBasedBuilder):
+
+    BUILDER_CONFIGS = [
+        MsMarcoConfig(
+            name="v1.1",
+            description="""version v1.1""",
+            version=datasets.Version("1.1.0", ""),
+        )
```
Comment (albertvillanova) [inline datasets/mmarco/mmarco.py:65]: You can use `datasets.BuilderConfig` instead of custom `MsMarcoConfig`.

**Review by albertvillanova — CHANGES_REQUESTED** [review]
Thanks @lhbonifacio for adding this dataset.

It is missing the dataset card, with all relevant information and metadata about the dataset. You can find:
- a template: https://github.com/huggingface/datasets/blob/master/templates/README.md
-  instructions to fill it:
  - https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card
  - https://github.com/huggingface/datasets/blob/master/templates/README_guide.md
-  and a complete example: https://github.com/huggingface/datasets/tree/master/datasets/eli5

**Comment by hugoabonizio** [comment]
Hi @albertvillanova we've made a major overhaul of the loading script including all configurations we're making available. Could you please review it again?

**Comment by hugoabonizio** [comment]
@albertvillanova :ping_pong:

**Comment by catqaq** [comment]
Hi there, i got an error about mmarco:
ConnectionError: Couldn't reach 'unicamp-dl/mmarco' on the Hub (ConnectionError)
code:
`from datasets import list_datasets, load_dataset
dataset = load_dataset('unicamp-dl/mmarco', language='portuguese')`

Any help will be appreciated!

**Comment by hugoabonizio** [comment]
Hi @catqaq, we updated the loading script. Now you can load the datasets with:

```python
dataset = load_dataset('unicamp-dl/mmarco', 'portuguese')
```

You can check the list of supported languages and usage examples in [this link](https://huggingface.co/datasets/unicamp-dl/mmarco). Feel free to contact us if you have any issues.

**Comment by catqaq** [comment]
> 



> Hi @catqaq, we updated the loading script. Now you can load the datasets with:
> 
> ```python
> dataset = load_dataset('unicamp-dl/mmarco', 'portuguese')
> ```
> 
> You can check the list of supported languages and usage examples in [this link](https://huggingface.co/datasets/unicamp-dl/mmarco). Feel free to contact us if you have any issues.

Thanks for your quick updates. So, how can i get the fixed version, install from the source?

**Comment by hugoabonizio** [comment]
@catqaq you can load mMARCO using the namespace `unicamp-dl/mmarco` while this PR remains under review.

**Comment by albertvillanova** [comment]
Thanks for your contribution, @lhbonifacio and  @hugoabonizio. And sorry for the late response.

We are removing the dataset scripts from this GitHub repo and moving them to the Hugging Face Hub: https://huggingface.co/datasets

As you already created this dataset under your organization namespace (https://huggingface.co/datasets/unicamp-dl/mmarco), I think we can safely close this PR.

We would suggest you complete your dataset card with the YAML tags, to make it searchable and discoverable.

Please, feel free to tell us if you need some help.
